ext/standard/ftp_fopen_wrapper.c

```diff
@@ -430,6 +430,7 @@ php_stream * php_stream_url_wrap_ftp(php_stream_wrapper *wrapper, const char *pa
 	int8_t read_write = 0;
 	char *transport;
 	int transport_len;
+	zend_string *error_message;
```

Comment:
Looks like `php_stream_xport_create()` does not NULL this variable out, so it will need a NULL initialization here.

Reply:
Even if php_stream_xport_create() did the null initialization, we'd still need the =NULL here, because an error might occur prior to the php_stream_xport_create() call. An alternative would be to move the error_string handling directly into the failure branch of the php_stream_xport_create() call, rather than having it in errexit. Either way should be fine.

Reply:
That's correct, NULL initialization is required. That has been fixed now.